public/doc/openapi-3-v1.2.json

```diff
@@ -3718,7 +3718,7 @@
             "type": "string",
             "title": "Scope of when the order item is ran",
             "example": "before",
-            "description": "Denotes the scope in which the order item will run for the order it belongs to. It can be 'before', 'after', or 'applicable'",
+            "description": "Denotes the scope in which the order item will run for the order it belongs to. It can be 'before', 'after', or 'product'",
```

Comment:
@hsong-rh please make the same change to your PR for v1.3.json

Reply:
Yes, I changed it already